packages/slice-machine/components/DeleteDocumentsDrawer/HardDeleteDocumentsDrawer.tsx

```diff
@@ -45,7 +47,7 @@ export const HardDeleteDocumentsDrawer: React.FunctionComponent = () => {
           variant="primary"
           onClick={() => {
             closeModals();
-            pushChanges();
+            pushChanges(true);
```

Comment:
hmmm ok, I wasn't sure.
I just assumed that the action of pushing changes in case of a hard delete (that require manual deletion of documents)
Should be treated as a true by default because in facts, documents have been deleted for this.

Reply:
If you clarified with Come then i'll change it to false, thank for telling me 😄

Reply:
ahhhh how dumb of me yeah you're totally right we don't want to assume they actually removed all the documents.